ADBDEV-251 Remove useless systemd parameter

acmnu opened the pull request:
There is a trouble with staring service under root,
while systemV is not woring with root.

acmnu committed:
ADBDEV-251 Remove useless systemd parameter

There is a trouble with staring service under root,
while systemV is not woring with root.

acmnu commented:
@Stolb27 Please check that one. Is this ok for us?

Stolb27 commented on bigtop-packages/src/common/pxf/pxf.service:
```diff
@@ -12,7 +12,6 @@ ExecStop=/usr/lib/pxf/bin/pxf stop
 ExecReload=/bin/kill -HUP $MAINPID
```
SIGHUP kill tomcat:) Remove it, please.

acmnu replied:
Done

acmnu committed:
ADBDEV-251 Remove reload command for pxf

According to ivi@arenadata.io SIGHUP kills PXF, so
we should not used that as reload command.
